tests/test_point.py

```diff
@@ -2,6 +2,11 @@
 from shapely.geometry import Point, asPoint
 from shapely.errors import DimensionError
 
+import pytest
+
+
+shapely20_deprecated = pytest.mark.filterwarnings("ignore:Setting:FutureWarning")
```

Comment:
Having written a couple of these `mark`ers for specifying required GEOS versions, I was wondering if we should put them in a common place (even just in `conftest.py`) so they don't need to be re-declared in every file in which they're used. Especially for one like this, which will likely get re-used many times in the coming months.

Reply:
It was lost on me at the time of commenting that the `:Setting:` part in the middle is referencing the text of the deprecation warning. Should there be a standard message that can be used everywhere deprecations are occurring?

Reply:
> Should there be a standard message that can be used everywhere deprecations are occurring?

Yes, that sounds a good idea to have all messages that will be related to "Shapely 2.0 deprecations" have a consistent structure. That makes filtering them out like the above easier with a single pytest marker, and is also more recognizable for users.

(we could actually also go for a warning subclass, for example sphinx has "RemovedInSphinx4Warning": https://github.com/sphinx-doc/sphinx/blob/776ec7957c3407a5ed75dc4369a0042d91698bd1/sphinx/deprecation.py#L21, but that can also easily be adopted later if we want to do so)

> I was wondering if we should put them in a common place (like conftest.py) so they don't need to be re-declared in every file in which they're used. 

Yes, certainly! It was just because I was only editing a single file here, but once I do the same for the other geometry classes, we certainly need to move it to a general place (and conftest.py seems appropriate for that with pytest)

Reply:
I quite like the idea of a `RemovedInShapely20` warning subclass. Would make it much easier for users to filter out all of those warnings with a single go, if they'd like to.

Reply:
I should have scrolled down here before I recommended the new class above :)